src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -494,6 +511,80 @@ struct aie_reg_read
   }
 };
 
+static std::unique_ptr<drm_fd> aie_get_drmfd(const xrt_core::device* device, std::string dev_path)
```

Comment:
Newline before function name; it's close to impossible to scan files that don't line up functions on the left.
const std::string&